SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/BrokerProfileJob.swift

```diff
@@ -127,7 +127,7 @@ public class DataBrokerOperation: Operation, @unchecked Sendable {
         let allBrokerProfileQueryData: [BrokerProfileQueryData]
```

Comment:
Github won't let me comment on the line above since it's not in the diff, but this function is still called runOperation. This I arguably fine since it's actually an operation, but I think could get confusing with the existence of start() too, so lean towards renaming it runJob